### DISCUSSION TITLE
Is there a way to permanently hide the menu bar?

### DISCUSSION THREAD

**Discussion by Kommynct** [zim-desktop-wiki/zim-desktop-wiki#2680] [Q&A] [answered]
I'd like to have a keybind for hiding the menu bar, I almost never use it and it looks ugly.

on gnome it's standard for alt+m to hide it.

**Comment by jaap-karssenberg** [comment]
I think there is a hidden key for this in either the preferences or the UI
state config file. Can't check right now.

Op za 21 sep. 2024 00:55 schreef Communist ***@***.***>:

**Reply by Kommynct** [reply]
i looked through preferences.conf and style.conf and didn't see anything. I couldn't find anything in the docs either.

**Comment by jaap-karssenberg** [comment]
Checked the code now. There is an "toggle_menubar" action method defined in
the MainWindow class. But currently it is only called from the distraction
free mode plugin.

Would be an easy patch to add a preference that triggers it.

Or create an almost empty plugin that does nothing else but call this
method.

Regards,

Jaap

Op za 21 sep. 2024 08:16 schreef Communist ***@***.***>:

**Comment by rui-nibau** [comment, marked as answer]
Hello.

here is a little plugin to that : #2896

* [Website](https://omacronides.com/projets/zim-plugin-hide-menubar)
* [Sources](https://framagit.org/rui.nibau/zim-plugin-hide-menubar)
